windows/internal/vc_install_helper.bat

```diff
@@ -1,7 +1,17 @@
+if "%VC_YEAR%" == "2017" powershell windows/internal/vs2017_install.ps1
```

Comment:
I don't think VS2017 is used anywhere now. We should probably keep 2 versions 2022 and 2019 (if we will need to fallback/revert) .  In this PR we should not add additional cases for VS2017. Please remove them. I think follow up PR to remove all the VS2017 artifacts will be a good idea.

Reply:
I've removed the VS2017 installation from the build scripts.